web/src/components/ExportEventButton.tsx

```diff
@@ -13,6 +13,11 @@ const ButtonContainer = styled.div`
   display: flex;
   gap: 16px;
 `
+const StyledButton = styled(Button)`
+  &:hover {
+    background-color: ${props => props.theme.palette.primary.light};
```

Comment:
💭 I'm not sure this is necessary but I also just realized that this button in our design doesn't match the ones from the design system, I'm checking with Toni in Figma what to do about this button